lib/configs/jsx.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+// @ts-ignore
+const react = require('eslint-plugin-react')
+
+module.exports.jsx = /** @satisfies {import('eslint').Linter.Config} */ ({
+  name: 'neostandard/jsx',
+
+  languageOptions: {
+    parserOptions: {
+      ecmaFeatures: {
+        jsx: true,
```

Comment:
The base configuration contains a `parserOptions` with comment out jsx enablement that can removed.

Reply:
Thanks! 🙏  Fixed